networkx/drawing/tests/test_pylab.py

```diff
@@ -18,6 +18,14 @@
 barbell = nx.barbell_graph(4, 6)
 
 
+@pytest.fixture
+def subplots():
+    fig, ax = plt.subplots()
+    yield fig, ax
+    plt.delaxes(ax)
+    plt.close()
```

Comment:
Hmm, any idea why this was failing with pytest-mpl?
```
Error: Image dimensions did not match.
                      Expected shape: (600, 800)
                        /home/circleci/project/results/networkx.drawing.tests.test_pylab.test_house_with_colors/baseline.png
                      Actual shape: (480, 640)
                        /home/circleci/project/results/networkx.drawing.tests.test_pylab.test_house_with_colors/result.png
```

Maybe we change the baseline image to make the fixture happy?

Reply:
Yeah I wasn't sure, so I did the simple thing and decided not to use the fixture in the `pytest-mpl` test, happy to crank on it more though if we want to try to make it work!